packages/whats-new/package.json

```diff
@@ -40,17 +39,16 @@
 		"@automattic/calypso-analytics": "^1.0.0-alpha.1",
 		"@babel/runtime": "^7.12.5",
 		"@automattic/react-i18n": "^1.0.0-alpha.0",
+		"@automattic/i18n-utils": "^1.0.0",
 		"@wordpress/components": "^12.0.7",
 		"wpcom": "^5.4.2"
 	},
 	"peerDependencies": {
-		"react": "^16.8",
-		"react-dom": "^16.8"
+		"react": "^16.8"
 	},
 	"devDependencies": {
 		"wpcom-proxy-request": "^6.0.0",
 		"@automattic/calypso-build": "^7.0.0",
-		"react": "^16.12.0",
-		"react-dom": "^16.12.0"
+		"react": "^16.12.0"
```

Comment:
Why is `react` duplicated in peer and dev dependencies? Is this just a pattern I'm not familiar with?

Reply:
Nope, just me not really understanding how the dependencies work :)